toolbox/pkgs/macos-12-build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# This is done to avoid appleframework deprecation warnings
+export MACOSX_DEPLOYMENT_TARGET=12.0
```

Comment:
This should be ``11.0`` to tell the toolchain that the code follows the old API, otherwise it will yield deprecation warnings here.